include/zenoh-pico/transport/multicast.h

```diff
@@ -0,0 +1,25 @@
+//
+// Copyright (c) 2022 ZettaScale Technology
+//
+// This program and the accompanying materials are made available under the
+// terms of the Eclipse Public License 2.0 which is available at
+// http://www.eclipse.org/legal/epl-2.0, or the Apache License, Version 2.0
+// which is available at https://www.apache.org/licenses/LICENSE-2.0.
+//
+// SPDX-License-Identifier: EPL-2.0 OR Apache-2.0
+//
+// Contributors:
+//   ZettaScale Zenoh Team, <zenoh@zettascale.tech>
+//
+
+#ifndef ZENOH_PICO_MULTICAST_H
+#define ZENOH_PICO_MULTICAST_H
+
+#include "zenoh-pico/api/types.h"
+
+bool _zp_is_multicast_here(void);
```

Comment:
What's the purpose of this function?

Reply:
It's used on the scouting function it replaces a `#if` that was doing the same.